### ISSUE TITLE
Install script

### ISSUE THREAD

**Issue by sillen102** [leftwm/leftwm#139]
Perhaps you have already considered this but an install script would have been nice for building from the github repo.

I made a simple one that looks like this:

```
#!/bin/sh
cargo build
mkdir -p /usr/share/leftwm
cp -r ./target/debug/* /usr/share/leftwm/
ln -s /usr/share/leftwm/leftwm /usr/bin/leftwm
ln -s /usr/share/leftwm/leftwm-worker /usr/bin/leftwm-worker
ln -s /usr/share/leftwm/leftwm-state /usr/bin/leftwm-state
cp leftwm.desktop /usr/share/xsessions/
mkdir -p $HOME/.config/leftwm/themes
cp -r themes/* $HOME/.config/leftwm/themes/
rm -r target
```

**Comment by cjgratacos** [comment]
@sillen102 that is a good idea but instead of using the debug binary, I would highly recommend using the release build. Just change: `cargo build` to `cargo build --release` and change the line `cp -r ./target/debug/* /usr/share/leftwm/` to `cp -r ./target/release/* /usr/share/leftwm/` and it will looks something like this

```rust
#!/bin/sh
cargo build --release
mkdir -p /usr/share/leftwm
cp -r ./target/release/* /usr/share/leftwm/
ln -s /usr/share/leftwm/leftwm /usr/bin/leftwm
ln -s /usr/share/leftwm/leftwm-worker /usr/bin/leftwm-worker
ln -s /usr/share/leftwm/leftwm-state /usr/bin/leftwm-state
cp leftwm.desktop /usr/share/xsessions/
mkdir -p $HOME/.config/leftwm/themes
cp -r themes/* $HOME/.config/leftwm/themes/
rm -r target

```

**Comment by edneville** [comment]
Maybe use /usr/local/leftwm rather than /usr/share/leftwm which is not distro managed.

**Comment by hertg** [comment]
Closed with #533
